templates/Dataland/datasetlinks.html.twig

```diff
@@ -0,0 +1,15 @@
+<h2>Dataset Links:</h2>
+<div>
+    {% set linkcodes = constant('App\\Entity\\DatasetLink::ONLINE_FUNCTION_CODES') %}
+    {% for link in dataset.datasetSubmission.datasetLinks %}
+        <p>
+            <b>Name:</b>{{ link.name }}<br>
+            <b>Description:</b>{{ link.description }}<br>
+            <b>Function:</b>{{ attribute(linkcodes[link.functionCode], 'name') }}<br>

```

Comment:
Should we add the same style as summary tab?

Reply:
Which style do you suggest? I tried h2, didn't look right. And the new lines use too much space I think.

Reply:
We can ask the PO because it is a public facing feature